src/app/components/challenge/challengephases/editphasemodal/editphasemodal.component.html

```diff
@@ -0,0 +1,116 @@
+            <div class="editor-validation-message fs-12"
+              *ngIf="isEditorFieldMessage">{{editorValidationMessage}}</div>
+            <div class="mt-20 mb-20 edit-phase-editor" [froalaEditor]
+              [(froalaModel)]="description"></div>
+          </div>
+        </div>
+      </div>
+
+      <div class="row row-lr-margin"></div>
+
+      <div class="row row-lr-margin">
+        <div class="col-lg-4 col-md-4 col-sm-12 col-lr-pad">
+          <div class="input-field align-left">
+            <span class="text-med-black fw-regular fs-16">Start date and time</span>
+            <app-input class="mt-20" [mindatetime]="todayDateTime"
+              [editPhaseDetails]="editPhaseDetails" [name]="'start_date'"
+              [isRequired]="true" [label]="'start_date'" [placeholder]="''"
+              [type]="'datetime'" [value]="startDate" #formmodal></app-input>
+          </div>
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-12 col-lr-pad">
+          <div class="input-field align-left">
+            <span class="text-med-black fw-regular fs-16">End date and time</span>
+            <app-input class="mt-20" [mindatetime]="startDate"
+              [editPhaseDetails]="editPhaseDetails" [name]="'end_date'"
+              [isRequired]="true" [label]="'end_date'" [placeholder]="''"
+              [type]="'datetime'" [value]="endDate" #formmodal></app-input>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-lg-4 col-md-4 col-sm-12 col-lr-pad">
+        <div class="input-field align-left">
+          <span class="text-med-black fw-regular fs-16">Submissions/day</span>
+          <app-input class="mt-20" id="max_submissions_per_day"
+            [editPhaseDetails]="editPhaseDetails"
+            [name]="'max_submissions_per_day'" [isRequired]="true"
+            [label]="'max_submissions_per_day'" [placeholder]="''"
+            [type]="'number'" [value]="maxSubmissionsPerDay" #formmodal></app-input>
+        </div>
+      </div>
+
+      <div class="col-lg-4 col-md-4 col-sm-12 col-lr-pad">
+        <div class="input-field align-left">
+          <span class="text-med-black fw-regular fs-16">Submissions/month</span>
+          <app-input class="mt-20" min="challenge_phase.max_submissions_per_day"
+            [editPhaseDetails]="editPhaseDetails" id="max_submissions"
+            [name]="'max_submissions'" [isRequired]="true"
+            [label]="'max_submissions_per_month'" [placeholder]="''"
+            [type]="'number'" [value]="maxSubmissionsPerMonth" #formmodal></app-input>
+        </div>
+      </div>
+
+      <div class="col-lg-4 col-md-4 col-sm-12 col-lr-pad">
+        <div class="input-field align-left">
+          <span class="text-med-black fw-regular fs-16">Total submissions</span>
+          <app-input class="mt-20"
+            min="challenge_phase.max_submissions_per_month"
+            [editPhaseDetails]="editPhaseDetails" id="max_submissions"
```

Comment:
What's `id="max_submissions"` is doing here? `id` is not there in `input.component.ts`. Is it for DOM?

Reply:
@lunayach well it's of no use, we can remove that